client/state/login/reducer.js

```diff
@@ -25,6 +33,21 @@ export const isRequesting = createReducer( false, {
 	[ LOGIN_REQUEST_SUCCESS ]: () => false,
 } );
 
+export const redirectTo = createReducer( null, {
```

Comment:
Do we need a new reducer for this? The data we need is already available in the state. Why not query it in the selector instead? (same question for rememberMe)